packages/agents/router/src/lib/operations/execute.ts

```diff
@@ -7,12 +7,29 @@ import {
   OriginTransfer,
 } from "@connext/nxtp-utils";
 
-import { CallDataForNonContract, MissingXCall, NotEnoughAmount, ParamsInvalid } from "../errors";
+import { MissingXCall, NomadHomeBlacklisted, NotEnoughAmount, ParamsInvalid } from "../errors";
 import { getHelpers } from "../helpers";
 import { getContext } from "../../router";
 
+import { NomadContext } from "@nomad-xyz/sdk";
+import { BridgeContext } from "@nomad-xyz/sdk-bridge";
+
 // fee percentage paid to relayer. need to be updated later
 export const RELAYER_FEE_PERCENTAGE = "1"; //  1%
+//helper function to match our config environments with nomads
+const nxtpEnvToNomadEnv = (envOverride?: string): string => {
+  const nxtpConfig = envOverride ? envOverride : getContext().config.environment;
+  switch (nxtpConfig) {
+    case "production":
+      return "production";
```

Comment:
These don't map directly to our env. We use production for testnets also. We need to have a separate config item for `nomadEnv`.